app_feup/lib/view/Theme.dart

```diff
@@ -18,6 +18,7 @@ ThemeData applicationTheme = new ThemeData(
   primaryColor: primaryColor,
   accentColor: accentColor,
   hintColor: hintColor,
+  indicatorColor: lightGreyTextColor,
```

Comment:
To follow the same standard, please put this inside the TextTheme data :smiley_cat: